handlers/handlers.go

```diff
@@ -0,0 +1,67 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/Daniorocket/cookit-backend/lib"
+	"github.com/gorilla/mux"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type Handler struct {
+	Client       *mongo.Client
+	DatabaseName string
+}
+type JWT struct {
+	Token    string
+	Username string
+}
+type Pagination struct {
```

Comment:
Wygląda że używasz tego tylko do responsów, to lepiej będzie`PaginationResponse`